x-pack/libbeat/management/managerV2.go

```diff
@@ -359,6 +377,8 @@ func (cm *BeatV2Manager) handleOutputReload(unit *client.Unit) {
 		_ = unit.UpdateState(client.UnitStateFailed, errString.Error(), nil)
 		return
 	}
+	// set to true, we'll reload the output if we need to re-configure
+	cm.outputIsConfigured = true
```

Comment:
This is here as a guard against restarting the first time the output is ever configured?

Reply:
yup.